lib/src/index.ts

```diff
@@ -17,9 +17,15 @@ export { default as NxModal, Props as NxModalProps } from './components/NxModal/
 export { default as NxTextInput, PublicProps as NxTextInputProps, StateProps as NxTextInputStateProps }
   from './components/NxTextInput/NxTextInput';
 
+export { default as NxDateInput, PublicProps as NxDateInputProps, StateProps as NxDateInputStateProps }
+  from './components/NxDateInput/NxDateInput';
+
 import * as nxTextInputStateHelpers from './components/NxTextInput/stateHelpers';
 export { nxTextInputStateHelpers };
 
+import * as nxDateInputStateHelpers from './components/NxTextInput/stateHelpers';
+export { nxDateInputStateHelpers };
```

Comment:
I'm debating whether we need this on a separate alias like this.  I guess it's more future-proof: even though at the moment its the same as NxTextInputStateHelpers, in the future it might not be.  It looks like the examples don't actually use this though - they should.

Reply:
> I'm debating whether we need this on a separate alias like this. I guess it's more future-proof: even though at the moment its the same as NxTextInputStateHelpers, in the future it might not be. It looks like the examples don't actually use this though - they should.

I missed that one: fixed: 91f1e2e204fc37deb462d6c54c9f2f26af0a8b8e